### DISCUSSION TITLE
What to do if explorer.exe restart during session ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [midrare/hyperenable#14] [Q&A] [answered]
Hi,

I’m not sure but it seems that if for some reason explorer.exe is restarted, it re-enables the office keys. Currently, I need to log out/log in to trigger hyperenable so it can do the work.

Would there be an alternative to apply hyperenable on a running system where explorer has restarted ?

**Comment by midrare** [comment]
Try starting `hyperenable.exe` then start `explorer.exe`. Just curious, what's your use case that needs you to restart `explorer.exe`?

**Comment by Fred-Vatin** [comment]
It is very rare I need it but sometimes, restarting explorer fixes some issues with the taskbar. It’s faster than logging out/logging in.

I tried this to no avail:
- kill `explorer.exe`
- start `hyperenable.exe` as admin (or not)
- start `explorer.exe`

When running `hyperenable.exe` through cmd, I get `Unrecognized subcommand`

![image](https://github.com/midrare/hyperenable/assets/22337329/43003263-ca8a-4ec8-94ab-10840e6c8259)

[Here](https://justpaste.it/eoxs7) are some details. The process lasts less than 500ms before exit.

**Reply by midrare** [reply]
Try `"c:\program files\hyperenable\hyperenable.exe"` or `.\hyperenable.exe`

**Comment by Fred-Vatin** [comment]
Same.

**Comment by midrare** [comment, marked as answer]
What about `hyperenable.exe start`?

**Comment by Fred-Vatin** [comment]
Much better. Thanks. I suggest this to be mentioned in the readme.
![image](https://github.com/midrare/hyperenable/assets/22337329/0d9a82d4-64d4-4d66-84da-882303ffa829)
